pkg/genericapiserver/config.go

```diff
@@ -440,3 +443,10 @@ func DefaultAndValidateRunOptions(options *options.ServerRunOptions) {
 		}
 	}
 }
+
+func NewRequestInfoResolver(c *Config) *request.RequestInfoResolver {
+	return &request.RequestInfoResolver{
+		APIPrefixes:          sets.NewString(strings.Trim(c.APIPrefix, "/"), strings.Trim(c.APIGroupPrefix, "/")), // all possible API prefixes
+		GrouplessAPIPrefixes: sets.NewString(strings.Trim(c.APIPrefix, "/")),                                      // APIPrefixes that won't have groups (legacy)
```

Comment:
Certainly true. But it's only moved code here. I have added a note to https://github.com/kubernetes/kubernetes/issues/32792 for a follow-up.